promql/aggregate_expr.go

```diff
@@ -17,7 +17,8 @@ const (
 )
 
 type aggregateFn struct {
-	name      string
+	name string
+	// packageName string
```

Comment:
I guess this is left over from other work, and you can remove it.

Reply:
Oh, I thought I deleted that. 🤦🏾‍♀️